src/main/java/com/notnoop/apns/PayloadBuilder.java

```diff
@@ -263,11 +264,24 @@ public PayloadBuilder instantDeliveryOrSilentNotification() {
     }
 
     /**
-     * Set the notification localized key for the alert body
-     * message.
+     * In order for the push notification to get picked up by UNNotificationServiceExtension, the
+     * aps dictionary must include 'mutable-content' key with value set to 1.
      *
-     * @param key   the localizable message body key
-     * @return  this
+     * @see https://developer.apple.com/reference/usernotifications/unnotificationserviceextension
+     *
+     * @return this
+     */
+    public PayloadBuilder mutableContent() {
+        aps.put("mutable-content", 1);
+        return this;
+    }
+
+    /**
```

Comment:
Looks like it's just auto-formatted the javadoc of the .method below. I'll clean it up in a bit. Thanks